### PR TITLE
To support an input for RG name

### PR DESCRIPTION
**Pull request by carpenike**
Our environment does not allow users to create new RGs. Therefore, the RG needs to exist and be provided as an input for the temporary storage account location.

### CODE REVIEW

**Comment by williexu** [comment]
@tamirkamara can you review?

**Comment by tamirkamara** [comment]
I'll will take care of this week

File: src/image-copy/azext_imagecopy/__init__.py [hunk 1]
```diff
@@ -46,6 +46,8 @@ def load_arguments(self, _):
             c.argument('timeout', options_list=['--timeout'], type=int, default=3600,
                        help='Time in seconds for the copy operation to finish. Increase this time if '
                        'you are going to copy large images (disks) like 512GB or more.')
+            c.argument('temporary_rg_name', options_list=['--temp-rg-name'], default='image-copy-rg',
```
Comment (tamirkamara) [inline src/image-copy/azext_imagecopy/__init__.py]: please rename the option and var name without abbreviations: **temporary_resource_group_name**

File: src/image-copy/azext_imagecopy/custom.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 # pylint: disable=too-many-statements
 # pylint: disable=too-many-locals
 def imagecopy(source_resource_group_name, source_object_name, target_location,
-              target_resource_group_name, source_type='image', cleanup='false',
+              temporary_rg_name, target_resource_group_name, source_type='image', cleanup='false',
```
Comment (tamirkamara) [inline src/image-copy/azext_imagecopy/custom.py]: 1. I think it makes sense to have both target parameters together, so please move the new _temporary_rg_name_ parameter after _target_resource_group_name_
1. Shouldn't the new parameter have a default value set here too, like all others have?

Comment (RyanHolt-Msft) [reply src/image-copy/azext_imagecopy/custom.py]: Sorry, not really sure what I did with 'approve these changes.' :) thought that was approving them for my PR. 

I believe it should have a default value as well as the other ones. Do I need to make these changes within my source and re-submit the PR or can it be changed within this view directly?

Comment (tamirkamara) [reply src/image-copy/azext_imagecopy/custom.py]: You should make changes in your source code and push to you fork. The PR will be updated automatically without the need to re-submit it.

Comment (tamirkamara) [reply src/image-copy/azext_imagecopy/custom.py]: BTW, looks like you also need to do a merge with the latest changes on the master branch

**Comment by williexu** [comment]
@ryholt @carpenike ping for updates

**Comment by tamirkamara** [comment]
@ryholt I'm ok with your changes but the CI is failing and blocks my approval

**Comment by carpenike** [comment]
Yup going to try to get it fixed this morning. The lint is failing because the line is too long.

**Pull request merged**